interface.idl

```diff
@@ -116,5 +116,12 @@ namespace rtc {
 namespace time {
     rpc gettimeofday() -> (unixtime: uint64);
 }
-
+namespace shm {
+    rpc create(size : size) ->  (shm_id : int);
+    rpc map(shm_id : int, flag:int)   ->  (vaddr : vaddr);
```

Comment:
Please fix whitespaces: e.g., this line should be `(shm_id: int, flag: int) -> (vaddr: vaddr);`